### ISSUE TITLE
Compatability issue with Extension:CirrusSearch

### ISSUE THREAD

**Issue by nha4601** [HydraWiki/mediawiki-embedvideo#88]
Errors appeared while trying to configure CirrusSearch related to the paths for $wgFFmpegLocation and 
$wgFFprobeLocation (both contained spaces).

After disabling CirrusSearch the media player no longer shows and white space apears instead.

**Comment by Alexia** [comment]
Can you give more details such as example paths and how this relates to settings in CirrusSearch?

**Comment by nha4601** [comment]
Yes; after installing CirrusSearch, Elastica, and Elasticsearch, the README file for CirrusSearch provides the following steps to bootstrap the search index:

1)  php $MW_INSTALL_PATH/extensions/CirrusSearch/maintenance/updateSearchIndexConfig.php
2) php $MW_INSTALL_PATH/extensions/CirrusSearch/maintenance/forceSearchIndex.php --skipLinks --indexOnSkip
3) php $MW_INSTALL_PATH/extensions/CirrusSearch/maintenance/forceSearchIndex.php --skipParse

It was during step 2 that the console would print out status messages for mediawiki regarding pages being indexed. Many rows printed out errors instead stating that D:\Program is not a recognizible command or batch file. These errors contined to appear every time step 2 was performed.

I couldn't get CirrusSearch working so I then disabled it in LocalSettings.php; however, all of my EmbedVideo players are not showing. I simply see white space instead.

If I disable EmbedVideo and re-run index step 2, then it completes without error.
